src/App/Controls/CircularProgressbarView.cs

```diff
@@ -21,13 +21,13 @@ public class CircularProgressbarView : SKCanvasView
             nameof(StrokeWidth), typeof(float), typeof(CircularProgressbarView), 3f);
 
         public static readonly BindableProperty ProgressColorProperty = BindableProperty.Create(
-            nameof(ProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.Default);
+            nameof(ProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.FromHex("175DDC"));
 
         public static readonly BindableProperty EndingProgressColorProperty = BindableProperty.Create(
-            nameof(EndingProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.Default);
+            nameof(EndingProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.FromHex("dd4b39"));
```

Comment:
Define the Color on the styles and then use it here to avoid hardcoding and maybe this color can be reused elsewhere